sklearn/tests/test_calibration.py

```diff
@@ -109,19 +119,22 @@ def test_sample_weight():
         X[:n_samples], y[:n_samples], sample_weight[:n_samples]
     X_test = X[n_samples:]
 
-    for method in ['sigmoid', 'isotonic']:
-        base_estimator = LinearSVC(random_state=42)
-        calibrated_clf = CalibratedClassifierCV(base_estimator, method=method)
-        calibrated_clf.fit(X_train, y_train, sample_weight=sw_train)
-        probs_with_sw = calibrated_clf.predict_proba(X_test)
+    for ensemble in (True, False):
```

Comment:
we typically prefer using test parametrization instead of adding for-loops (this file is old, so we haven't parametrized it yet).

parametrizing this test would look like

```py
@pytest.mark.parametrize('ensemble', [True, False])
def test_sample_weight(ensemble): 
...
```

That would reduce the diff and help reviewing